aspnetcore/test/hot-reload.md

```diff
@@ -26,13 +26,45 @@ For more information on supported scenarios, see [Supported code changes (C# and
 
 ## Blazor WebAssembly
 
-Blazor WebAssembly Hot Reload support has the following conditions:
+::: moniker range=">= aspnetcore-7.0"
 
-* Hot Reload reacts to most changes to method bodies, such as adding, removing, and editing variables, expressions, and statements.
```

Comment:
The first two bullet points in this list are still valid. The list you added below is for the edits that we *added* support for in .NET 7.